src/dll/edbgEcmdDll.C

```diff
@@ -2030,6 +2045,13 @@ uint32_t dllIStepsByName(std::string i_stepName) {
        rc = g_edbgIPLTable.istepPowerOn();
        if (!rc)
        {
+           //Initialize the libekb
+           rc = libekb_init();
+           if (rc)
+           {
+               return out.error(rc, FUNCNAME,
+                               "libekb_init() failed\n");
+           }
```

Comment:
It looks like you need to refactor the above code into its own function, rather than having the same code repeated three times. Ie. Can you create a patch to move the existing ipl_init() code into a separate function then a second patch to add the libekb_init() call to that function. Thanks.

Reply:
Agreed , libekb_init() and ipl_init(IPL_HOSTBOOT) keep in same function.

Reply:
Done